pages/mercury_tour_page.py

```diff
@@ -1,73 +1,126 @@
 from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
 
 
 class BasePage(object):
```

Comment:
why we've made this `BasePage`?
this has the hardcoded url used in child class `MainWelcomePage`.

we can either remove this class entirely and define `__init__` and `visit` methods in child class OR keep this class abstract to get inherited by other classes.

Reply:
Though its an abstract class, but i am going to define __init__ and visit in same (MainWelcomePage) class